lib/p2p/Peer.ts

```diff
@@ -251,16 +251,8 @@ class Peer extends EventEmitter {
     }
   }
 
-  private increaseBan = (score: number): boolean => {
-    this.banScore += score;
-
-    if (this.banScore >= 100) { // TODO: make configurable
-      this.logger.debug(`Ban threshold exceeded (${this.nodePubKey})`);
-      this.emit('ban');
-      return true;
-    }
-
-    return false;
+  private increaseBan = (event: ReputationEvent) => {
```

Comment:
I think we can get rid of this method and just use `this.emit('reputationEvent', ...`. Or if you'd prefer to keep it let's rename it to something like `emitReputationEvent`, since `increaseBan` is less descriptive and may even be wrong if we ever emit positive reputation events from here.